README.md

```diff
@@ -39,21 +39,24 @@ You can create this file manually, or, use the `init` command to be guided throu
 
 ```bash
 $ pb init
+hostname? (optional) ******
 app_id? (required) *********
 user_key? (required) ********
 botname? (recommended) ********
-hostname? (optional) ******
 ```
 
 The hostname field is optional, and will default to `aiaas.pandorabots.com`.
 
-#### botkey feature for api.pandorabots.com users (optional)
-v1.3.0 allows users to talk to the bot using a public botkey in place of the user_key and app_id. 
-This feature is only for api.pandorabots.com users, and can be manually specified in chatbot.json as follows:
+#### botkey feature for api.pandorabots.com users
+
+v1.3.1 introduces new talk, atalk, chat and achat functionalities through `public botkey` for premium users i.e. users with the 
+`hostname: api.pandorabots.com `
```

Comment:
Suggested copy:

v1.3.1 introduces a new `public botkey` option for talk, atalk, chat, and achat interactions by premium users i.e. users of `api.pandorabots.com`

This is primarily used for added security in browser-based talk interactions, but is available in `pb-cli` for testing purposes.

You can add this manually in `chatbot.json`, or use the `init` command to be guided through the process: